apertium-init.py

```diff
@@ -149,10 +149,16 @@ def make_replacements(s, replacements, conditionals):  # type: (str, Dict[str, s
 
 def make_all_replacements(destination, files, replacements, conditionals):  # type: (str, Dict[str, bytes], Dict[str, str], List[str]) -> None
     for filename, encoded_file in files.items():
-        path = os.path.join(destination, make_replacements(filename, replacements, conditionals))
+        repl_fname = make_replacements(filename, replacements, conditionals)
```

Comment:
```suggestion
        replacements_fillename = make_replacements(filename, replacements, conditionals)
```
Has better consistency since it's spelled out everywhere else in this file.